src/sentry/static/sentry/app/utils/testablePose.tsx

```diff
@@ -1,14 +1,16 @@
 /* global process */
 
+type PoseConfig = {[key: string]: any};
```

Comment:
```suggestion
type PoseConfig = Record<string, any>;
```

We could use record for this:
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt

Reply:
TIL about `Record`.

Reply:
I'll convert some other stuff over to this too.